scene/register_scene_types.cpp

```diff
@@ -785,13 +785,13 @@ void register_scene_types() {
 	OS::get_singleton()->yield(); //may take time to init
 
 	for (int i = 0; i < 20; i++) {
-		GLOBAL_DEF("layer_names/2d_render/layer_" + itos(i + 1), "");
-		GLOBAL_DEF("layer_names/3d_render/layer_" + itos(i + 1), "");
+		GLOBAL_DEF(vformat("%s/layer_%d", PNAME("layer_names/2d_render"), i + 1), "");
```

Comment:
Shouldn't you localize `layer` too?

Reply:
Since it's the formated string (`layer_1` / `Layer 1`) that will be sent for localization. Translating `layer` alone won't work. It's currently a limitation.